tinyxml2.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=${prefix}
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
Technically this should be `${exec_prefix}/@CMAKE_INSTALL_LIBDIR@`, I think.

Reply:
thanks; I just made that change in e06a240

I also found a good explanation of why this matters in https://github.com/libuv/libuv/pull/993